client/src/sldsLanguageClient.ts

```diff
@@ -76,10 +77,31 @@ function createServerPromise(context: ExtensionContext, outputChannel: OutputCha
 	return new Promise((resolve, reject) => {
 		var server = net.createServer((socket) => {
 			outputChannel.appendLine("SLDS Started");
+			
+			// Temporary solution for an LWC plugin issue where the end character range is too large for SLDS LSP server.
+			let filteredDuplex = new class extends Transform {
+				_transform(chunk: any, encoding: string, callback: TransformCallback) {
+
+					let buf = Buffer.from(chunk).toString();
+					let matcher = /("character":1.7976931348623157e\+308)/;
```

Comment:
It's probably better to declare this line, and line 90 on the global level.

Reply:
Could you explain why you suggest this?

Reply:
@doniarobinson-sf - can you elaborate your question? I am not sure if you are asking regarding the regex, or the move to global. Thanks.

Reply:
That comment was inline in response to your global comment.

Reply:
These 2 lines are pretty much constant. It doesn't require any data within the loop to create the Regex. 

Once the code is bumped up to constant, it should use `const` instead of `let` to prevent changes.

Reply:
Thank you @billyma and @doniarobinson-sf. I have added this feedback to the latest commit.